src/sentry/api/serializers/rest_framework/rule.py

```diff
@@ -88,11 +88,6 @@ def validate_environment(self, environment):
 
         return environment
 
-    def validate_conditions(self, value):
-        if not value:
-            raise serializers.ValidationError(u"Must select a condition.")
-        return value
-
```

Comment:
We should probably validate that at least one condition or filter has been added to the rule.

Reply:
Hmmm i think we will allow users to create an alert rule with no conditions or filters, because with `the event occurs` condition disabled the only way to create a rule that activates on every event is by supplying no conditions or filters